terraform/terraform-create-resource/cosmosdb.tf

```diff
@@ -0,0 +1,43 @@
+# azurerm_cosmosdb_account.db:
+resource "azurerm_cosmosdb_account" "db" {
+    access_key_metadata_writes_enabled    = true
+    analytical_storage_enabled            = false
+    enable_automatic_failover             = false
+    # Only one free tier cosmosDB per subscription
+    # enable_free_tier                      = true
+    enable_multiple_write_locations       = false
+    is_virtual_network_filter_enabled     = false
+    kind                                  = "GlobalDocumentDB"
+    local_authentication_disabled         = false
```

Comment:
this is like double negative haha. I suppose there's no `local_authentication_enabled`?

Reply:
This is a fixed name for this argument. However, I found out it is optional and defaults to false. I will have it removed along with other optional arguments. Documentation can be found here: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account